script/common

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Sets common variables that can be reused in other scripts
+
+PACKAGE_PATH="github.com/github/gh-ost"
+GO_VERSION=$(cat ./GO_VERSION)
+
+RELEASE_VERSION=
+if [ -z "${RELEASE_VERSION}" ] ; then
+  RELEASE_VERSION=$(git describe --abbrev=0 --tags | tr -d 'v')
+fi
+if [ -z "${RELEASE_VERSION}" ] ; then
+  RELEASE_VERSION=$(cat RELEASE_VERSION)
+fi
+
+DGO="docker run -u $(id -u):$(id -g) --rm \
```

Comment:
`-u $(id -u):$(id -g)` I do this so that the user/group permissions of the files generated stay the host user's. Otherwise it becomes `root:root` and a permissions pain.